code/frameworks/ember/template/cli/Button.stories.js

```diff
@@ -2,7 +2,7 @@ import { hbs } from 'ember-cli-htmlbars';
 import { action } from '@storybook/addon-actions';
 import { linkTo } from '@storybook/addon-links';
 
-// More on how to set up stories at: https://storybook.js.org/docs/ember/writing-stories/introduction
+// More on how to set up stories at: https://storybook.js.org/docs/ember/writing-stories
```

Comment:
@kylegach quick question, do we need to preserve the Ember framework here? And if so, shouldn't the link below use it as well for consistency?

Reply:
Good catch! No, we do not. I simply didn't include `ember` links in my Find/Replace'ing.